testsuite/features/step_definitions/salt_steps.rb

```diff
@@ -426,7 +426,11 @@
 
 When(/^I enter the IP address of "([^"]*)" in (.*) field$/) do |host, field|
   node = get_target(host)
-  output, _code = node.run("ip address show dev eth0")
+  if host == 'ubuntu-minion'
+    output, _code = node.run("ip address show dev ens3")
+  else
+    output, _code = node.run("ip address show dev eth0")
+  end
```

Comment:
Shorten the five lines above with a ternary and interpolation, and remember to use the single quotes for non-variable strings.